### DISCUSSION TITLE
Entity Hider - Hide Random Event NPCs

### DISCUSSION THREAD

**Discussion by GETrackerDan** [runelite/runelite#15042] [Ideas]
Would it be possible to add a way to hide random event npcs?

I'm realising how annoying it is whilst cooking at Burthorpe when random events standing on the fire!

Using the 'HIde NPCs' option hides the banker otherwise i'd use that.

**Comment by kamielvf** [comment]
You have the option "show attackers" on. This will show anyone who is interacting with you (e.g. following like random events)

**Comment by GETrackerDan** [comment]
Unfortunately i don't have anything enabled.

![n113s2f](https://user-images.githubusercontent.com/38025439/39603722-5b3ed3ba-4f21-11e8-98ea-3b34bf781ae1.png)

**Comment by GETrackerDan** [comment]
Also,  it would need to hide other people's random event npc's otherwise pointless.

**Comment by deathbeam** [comment]
The feature is not made for purposes like this, so I disagree this should be implemented.

**Comment by GETrackerDan** [comment]
I don't see how it would be any different to hiding players? I usually have the hide players option enabled but what's the point when i get to see their random event's every so often when skillng.

**Comment by Z7-z** [comment]
Excellent idea. 4 years later lol

**Comment by Adam-** [comment]
added in f9a56c7cfe11de9a3b34ddaf7e69e3a81c3dba30
